renderer/reducers/peripherals.ts

```diff
@@ -1,4 +1,14 @@
-import { PeripheralTypes } from '../constants/Constants';
+import * as consts from '../consts';
```

Comment:
NOTE: This file is still not type-safe, but we think it's better to wait until Lawrence's PR (https://github.com/pioneers/dawn/pull/14) is merged in first before we add types here. Currently, this branch doesn't have the news protos yet, so it's hard to see the types.